### DISCUSSION TITLE
Using the framework for non-geographical data

### DISCUSSION THREAD

**Discussion by dgrospelier** [FlowmapBlue/FlowmapBlue#74] [General]
Hi,
I'm looking for a solution to represent a flow map but flows are not related to geographical locations. The flows are data going out or coming into applications inside a company information system. 
I think I can use your tool for this graphical representation, but I need to remove the geographical part, do you think it could be possible or maybe could you please share ideas to do this if your solution is not relevant to my needs?
Thanks.

**Comment by ilyabo** [comment]
You could run some graph layout algorithm e.g. d3 force or use a tool like gephi, then extract the coordinates and remap them so that they are within -180 to 180 for longitude and -90 to 90 for latitude.
